Documentation/gettingstarted/bird.rst

```diff
@@ -0,0 +1,294 @@
+(O'Reilly, 2017) for a quick start.
+
+In the following, we will restrict our BGP scenario as follows:
+
+.. raw:: html
+
+    <p align="center"><img src="images/bird_simple_scheme.png" width="35%" height="35%"></p>
+
+* physical network: simple 3-tier hierarchical network
+* nodes connect to physical network via layer 2 switches
+* announcing each node's PodCIDR to physical network via bird
+* for each node, do not import route announcements from physical network
+
+This scheme is simple in that:
+
+* physical network could learn all the PodCIDRs from bird, which makes the Pod
+  IP addresses routable within the entire network.
+* each node doesn't need to learn and install routes from physical network (it
+  just sends pod traffic to node's default gateway, which is configured on the
+  physical network, and let the latter do the routing), which keeps the routing
+  table in each node clean and small, and suffers no performance issue.
+
+Below is the a reference configuration for implementing the above scheme:
+
+.. code:: bash
+
+    $ cat /etc/bird.conf
+    log syslog all;
+    
+    router id {{ NODE_IP }};
+    
+    protocol device {
+            scan time 10;           # Scan interfaces every 10 seconds
+    }
+    
+    # Disable automatically generating direct routes to all network interfaces.
+    protocol direct {
+            disabled;               # Disable by default
+    }
+    
+    # Forbide synchronizing BIRD routing tables with the OS kernel.
+    protocol kernel {
+            ipv4 {                    # Connect protocol to IPv4 table by channel
+                    import none;      # Import to table, default is import all
+                    export none;      # Export to protocol. default is export none
+            };
+    }
+    
+    # Static IPv4 routes.
+    protocol static {
+          ipv4;
+          route {{ POD_CIDR }} via "cilium_host";
+    }
+    
+    # BGP peers
+    protocol bgp uplink0 {
+          description "BGP uplink 0";
+          local {{ NODE_IP }} as {{ NODE_ASN  }};
+          neighbor {{ NEIGHBOR_0_IP }} as {{ NEIGHBOR_0_ASN }};
+          password {{ NEIGHBOR_PWD }};
+    
+          ipv4 {
+                  import filter {reject;};
+                  export filter {accept;};
+          };
+    }
+
+    protocol bgp uplink1 {
+          description "BGP uplink 1";
+          local {{ NODE_IP }} as {{ NODE_ASN  }};
+          neighbor {{ NEIGHBOR_1_IP }} as {{ NEIGHBOR_1_ASN }};
+          password {{ NEIGHBOR_PWD }};
+    
+          ipv4 {
+                  import filter {reject;};
+                  export filter {accept;};
+          };
+    }
+
+
+Replace the placeholders with your own ones:
+
+- ``NODE_IP``: IP address of this node
+- ``NODE_ASN``: autonomous system number (ASN) of this node
+- ``NEIGHBOR_*_IP``: IP address of the BGP peer
+- ``NEIGHBOR_*_ASN``: ASN of the BGP peer
+- ``NEIGHBOR_PWD``: password of the BGP peer
```

Comment:
If you are considering copy-paste-and-run by users, maybe i should rewrite this part as something like following?

```shell
sed -i 's/NODE_ASN/<node asn>/g' /etc/bird.conf
sed -i 's/NEIGHBOR_0_IP/<neighbor 0 ip>/g' /etc/bird.conf
sed -i 's/NEIGHBOR_1_IP/<neighbor 1 ip>/g' /etc/bird.conf
sed -i 's/NEIGHBOR_PWD/<neighbor password>/g' /etc/bird.conf
```

This may be more easier for new users.

Reply:
Ah, yes. That's better :+1:

Reply:
Thank you, i will update in this way.